src/style.css

```
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  border-radius: 0.5px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 div:nth-of-type(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--2 div:nth-of-type(-n+2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--3 div:nth-of-type(-n+3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--4 div:nth-of-type(-n+4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--5 div:nth-of-type(-n+5) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
```

_If several selectors MUST always have the same styles, group them using , to prevent accidental out ot sync in future_
And dont use **tags** here
No need to repeate styles, only image will change for this modifyers
```suggestion
.stars--1 div:nth-of-type(1) ,
.stars--2 div:nth-of-type(-n+2) ,
.stars--3 div:nth-of-type(-n+3) ,
.stars--4 div:nth-of-type(-n+4) ,
.stars--5 div:nth-of-type(-n+5) {
  background-image: url(images/star-active.svg);
}
```